frontend/src/pages/AnalyticsPage.js

```diff
@@ -1,132 +1,153 @@
-      error => this.setState({ mediaItem: null })
-    );
-  }
-
-  render() {
-    const { mediaItem } = this.state;
-    const { chartData, classes, match: { params: { pk } } } = this.props;
-
-    return (
-      <Page>
-        <BodySection classes={{ root: classes.section }}>
-          <Grid container spacing={16} >
-            <Grid item xs={12}>
-              <Typography variant="headline" component="div">
-                Viewing history (views per day)
-              </Typography>
-            </Grid>
-          </Grid>
-        </BodySection>
-        <BodySection classes={{ root: classes.section }}>
-          <div className={ classes.chartContainer }>
-            {
-              chartData.length > 1
-              ?
-              <Typography variant='body1' component='div'>
-                <Chart
-                  chartType="AnnotationChart"
-                  data={chartData}
-                  options={{fill: 100, colors: ['#EF2E31']}}
-                />
-              </Typography>
-              :
-              <Typography variant="subheading">
-                There is no data available for the media
-              </Typography>
-            }
-          </div>
-        </BodySection>
-        <BodySection classes={{ root: classes.section }}>
-          <Grid container spacing={16}>
-            <Grid item xs={12}>
-              <Typography variant="headline" component="div">
-                { mediaItem && mediaItem.name }
-              </Typography>
-            </Grid>
-          </Grid>
-          <Grid container justify='space-between' spacing={16}>
-            <Grid item xs={12} sm={6} md={3} lg={2}/>
-            <Grid item xs={12} sm={6} md={3} lg={2} style={{textAlign: 'right'}}>
-              {
-                mediaItem && mediaItem.legacyStatisticsUrl
-                ?
-                <Button component='a' variant='outlined' className={ classes.link } fullWidth
-                  href={mediaItem.legacyStatisticsUrl}
-                >
-                  SMS Statistics
-                  <ShowChartIcon className={ classes.rightIcon } />
-                </Button>
-                :
-                null
-              }
-            </Grid>
-          </Grid>
-        </BodySection>
-      </Page>
-    );
-  }
-}
-
-AnalyticsPage.propTypes = {
-  chartData: PropTypes.array.isRequired,
+/**
+ * The media item's analytics page contents
+ */
+const AnalyticsPageContents = ({
+  classes,
+  analytics: { viewsPerDay, size, totalViews },
+  resource: mediaItem
+}) => (
+  <div>
+    <BodySection classes={{ root: classes.section }}>
+      <Typography variant="headline" component="div">
+        { mediaItem && mediaItem.title }
+      </Typography>
+    </BodySection>
+    <BodySection classes={{ root: classes.section }}>
+      <Typography variant="title" component="div" className={ classes.subtitle } >
+        General Statistics
+      </Typography>
+      <Grid container>
```

Comment:
Could we have ``alignContent="center"``? I think it looks a bit weird on desktop all shoved over to the left.